tests/utils/mqtt_broker.py

```
@@ -0,0 +1,194 @@
+from __future__ import annotations
+import os
+import subprocess
+import logging.config
+import time
+import threading
+import sys
+
+sys.path.append(".")
+
+import external_server as _external_server
+from paho.mqtt.client import MQTTMessage as _MQTTMessage
```

Ruff

`paho.mqtt.client.MQTTMessage` imported but unused

Remove unused import: `paho.mqtt.client.MQTTMessage`

(F401)